academic-services/graph/resources-faq.md

```diff
@@ -59,4 +59,10 @@ It is relatively easy and inexpensive to compute citation counts and h-index usi
 
 ## What is FamilyId in Papers.txt
 
-Some papers were published in multiple venues, e.g. in archive and a conference. These papers share same FamilyId where the value  being the PaperId of the main paper.
+Some papers were published in multiple venues, e.g. in archive and a conference. These papers share same FamilyId where the value being the PaperId of the main paper.
```

Comment:
How about this for a definition?
"A paper "family" is defined as a group of papers that have been determined to be the same fundamental paper, but published in different venues (e.g. pre-print and conference). In this case only one of the papers will be determined to be the "primary" paper, and each paper in the family group will have its "FamilyId" value set to the primary papers ID.

If the "FamilyId" value is not defined for a paper it means that the paper is not part of a larger family group, however it is still considered to be the primary paper of a single-paper family."

Reply:
suggestion applied.